### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -35222,6 +35257,7 @@ wavelnes->wavelines
 wayoint->waypoint
 wayoints->waypoints
 wayword->wayward
+weahter->weather
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:35260]:
```text
weahters->weathers
```
https://github.com/search?q=weahters&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:35489]: [b6ce152](https://github.com/codespell-project/codespell/pull/2431/commits/b6ce1527bad1cb708dac6bb96ee09dd37465b509)

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -23434,6 +23455,10 @@ parms->params, prams,
 parmter->parameter
 parmters->parameters
 parnoia->paranoia
+parnter->partner
+parnters->partners
+parntership->partnership
+parnterships->partnerships
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:23653]: [b6ce152](https://github.com/codespell-project/codespell/pull/2431/commits/b6ce1527bad1cb708dac6bb96ee09dd37465b509)

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -17888,6 +17907,7 @@ infrantryman->infantryman
 infrasctructure->infrastructure
 infrastrcuture->infrastructure
 infrastruture->infrastructure
+infrastucture->infrastructure
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: [b6ce152](https://github.com/codespell-project/codespell/pull/2431/commits/b6ce1527bad1cb708dac6bb96ee09dd37465b509)

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -10147,6 +10159,9 @@ descover->discover
 descovered->discovered
 descovering->discovering
 descovery->discovery
+descrease->decrease
+descreased->decreased
+descreases->decreases
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:10239]: [b6ce152](https://github.com/codespell-project/codespell/pull/2431/commits/b6ce1527bad1cb708dac6bb96ee09dd37465b509)

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -2995,6 +2995,11 @@ assmumes->assumes
 assmuming->assuming
 assmumption->assumption
 assmumptions->assumptions
+assoaiate->associate
+assoaiated->associated
+assoaiates->associates
+assoaiation->association
+assoaiations->associations
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:3028]: [b6ce152](https://github.com/codespell-project/codespell/pull/2431/commits/b6ce1527bad1cb708dac6bb96ee09dd37465b509)

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**